tests/Traits/InvalidCodeAnalysisWithIssuesTestTrait.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Psalm\Tests\Traits;
+
+use Psalm\Config;
+use Psalm\Context;
+
+use function str_replace;
+use function strpos;
+use function strtoupper;
+use function substr;
+use function version_compare;
+
+use const PHP_OS;
+use const PHP_VERSION;
+
+/**
+ * Support for testing with the creation of a list of issues
+ *
+ *
+ * #### Description of motivation
+ *
+ * We have two different behaviors for the code related to
+ * "CodeException":
+ *
+ * ```
+ * $codebase->config->throw_exception = true; // or false
+ * ```
+ *
+ * If "throw_exception" is set to `true`, code execution stops.
+ *
+ * If "throw_exception" is set to `false`, the code may continue
+ * to execute, and an error may potentially occur.
```

Comment:
```suggestion
 * When `throw_exception` is set to `true`, code execution stops once 
 * the first issue is emitted, thus it may mask any problems after 
 * that point.  
 *
 * When `throw_exception` is set to `false`, the code will continue 
 * to be executed and we can uncover some additional bugs. 
```

Reply:
Thanks!